packages/dependency-royalty-management/contracts/RoyaltySplitter.sol

```diff
@@ -89,7 +89,7 @@ contract RoyaltySplitter is Initializable, OwnableUpgradeable, IRoyaltySplitter,
 
     /// @notice Splits and forwards ETH to the royalty receivers
     /// @dev normally ETH should be split automatically by receive function.
-    function splitETH() public {
+    function splitETH() public payable {
```

Comment:
Why does this need to be added now (how was it ok before)?

Reply:
Need this to check for covergae else ETH is automatically split.

Reply:
Sorry, what do you mean by this - eth is supposed to be automatically split

Reply:
@rishabh0x00